src/webview/helm-chart/app/cardItem.tsx

```diff
@@ -117,98 +118,162 @@ export class CardItem extends React.Component<DevFileProps, {
+                                        textAlign: 'left'
+                                    }
+                                }}
+                                error={installResponse.error}
+                                id='bootstrap-input'
+                                value={installChartName}
+                                sx={{
+                                    input: {
+                                        WebkitTextFillColor: this.props.themeKind <= 1 ? 'black' : 'white',
+                                        '&:disabled': {
+                                            WebkitTextFillColor: this.props.themeKind <= 1 ? 'black' : 'white'
                                         }
-                                    }} onChange={(e) => this.textFieldChange(e.target.value)}>
-                                </TextField>
-                            </div>
+                                    }
+                                }} onChange={(e) => this.textFieldChange(e.target.value)}>
+                            </TextField>
+                        </div>
+                        <div style={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', marginTop: '1rem' }}>
+                            <InputLabel
+                                style={{
+                                    color: 'var(--vscode-foreground)',
+                                    paddingTop: '0.5rem',
+                                    marginLeft: '1rem'
+                                }}>
+                                Version:
+                            </InputLabel>
+                            <Autocomplete
+                                id='grouped-folder'
+                                options={this.state.versions}
+                                autoHighlight
+                                fullWidth
+                                disableClearable
+                                disabled={this.state.versions?.length <= 1}
+                                value={selectedVersion}
+                                getOptionLabel={(option) => option.appVersion || option.version}
+                                renderInput={(params) => (
+                                    <TextField
+                                        sx={{
+                                            input: {
+                                                WebkitTextFillColor: this.props.themeKind <= 1 ? 'black' : 'white',
+                                                '&:disabled': {
+                                                    WebkitTextFillColor: this.props.themeKind <= 1 ? 'black' : 'white'
+                                                },
+                                                padding: '0.5rem !important'
+                                            }
+                                        }}
+                                        style={{ paddingTop: '10px', marginLeft: '0.5rem' }}
```

Comment:
```suggestion
                                        style={{ marginLeft: '10px
```suggestion
                                        style={{ paddingTop: '10px', marginLeft: '0.5rem' }}
```' }}
```